c/src/point_one/polaris/polaris.c

```diff
@@ -1385,7 +1385,7 @@ void P1_PrintData(const uint8_t* buffer, size_t length) {
     }
```

Comment:
this actually has a buffer overflow, we should check str_length is < 64 in this loop.